R/pagination.R

```diff
@@ -893,6 +893,28 @@ setGeneric("has_page_title", function(obj) standardGeneric("has_page_title"))
 
 setMethod("has_page_title", "ANY", function(obj) length(page_titles(obj)) > 0)
 
+# Retain listing key column formatting with pagination
+paginate_listing_strings <- function(df, i, mpf) {
+  l_mfs <- lapply(i, function(ii) {
+    mf_strings(matrix_form(df[ii, ]))
+  })
+
+  if (length(l_mfs) > 1) {
+    l_res <- lapply(l_mfs[-1], function(x) {
```

Comment:
This does not account for manual line breaks inserted in header. To be fixed...

Reply:
This should be resolved when matrix_form is called. The fact that is not done is a bug that I intend to fix asap